.ci_support/linux_64_.yaml

```diff
@@ -7,8 +7,8 @@ channel_targets:
 docker_image:
 - quay.io/condaforge/linux-anvil-x86_64:alma9
 pari:
-- 2.15.* *_pthread
+- 2.17.* *_pthread
 pyqt:
 - '5.15'
 python_min:
-- '3.10'
+- '3.9'
```

Comment:
napari is python 3.10+, we do not support python 3.9

Reply:
i think this is a rendering bug reported https://github.com/conda-forge/conda-smithy/issues/2316

the build uses 3.10 as can be seen in the logs

Reply:
So this value will appear but the actual build will use 3.10 due to the jinja override.

Reply:
See the build log here where it did use 3.10: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1239433&view=logs&jobId=7b6f2c87-f3a7-5133-8d84-7c03a75d9dfc&j=7b6f2c87-f3a7-5133-8d84-7c03a75d9dfc&t=9eb77fd2-8ddd-5444-8fc0-71cb28dcb736